src/main/java/com/cburch/logisim/fpga/data/MapComponent.java

```diff
@@ -678,7 +680,7 @@ public String getDisplayString(int pin) {
 
   public void getMapElement(Element Map) throws DOMException {
     if (!hasMap()) return;
-    Map.setAttribute(MAP_KEY, getDisplayString(-1));
+    Map.setAttribute(MAP_KEY, getDisplayString(-2));
```

Comment:
can we have magic value "-2" replaced with constant? If not or it makes no sense then please add the comment about why -2 is used here.

Reply:
The -2 has been replaced by ONLY_IO_MAP_NAME, done